README.md

```diff
@@ -59,11 +59,13 @@ brew outdated llvm || brew upgrade llvm
 mkdir build
 cd build
 cmake .. -DCMAKE_PREFIX_PATH=$(brew --prefix llvm)
-make install
+make -j install
```

Comment:
this  is not advisable, will crash people's computers under common circumstances.